appveyor.yml

```diff
@@ -1,7 +1,7 @@
 image: Visual Studio 2017
 
 environment:
-  nuget_version: '2.0.0-beta.1'
+  nuget_version: '2.0.0-beta.2'
   nuget_prerelease: false
```

Comment:
Would we want to mark this nuget as prerelease one?

Reply:
Yeah I think we should, just until we confirm there are no bugs in the refactor.

Reply:
Just as a heads up on the roadmap, I'm currently 75% done with one more refactor for the provider code, and then I'll be able to add the code to support dotnet core. So we'll likely have at least one more pre-release.